calculator-project/src/components/Keypad.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react'
+
+class Keypad extends Component {
+    render() {
+        return (
+            <div className="buttons">
+                <button name ="1" onClick={e => this.props.onClick(e.target.name)}>1</button>
```

Comment:
Instead of writing out all of this code for each button, you could create an array:

`const buttons = ["1", "2", ...,  "+/-", ....]`

Then in your render method, you could use map to create the buttons:

`{buttons.map(name => <button key={name} name={name} onClick={...} >{name}</button>)}`

This keeps your code more DRY!